vendor/vendor.json

```diff
@@ -32,54 +32,6 @@
 			"revision": "e2ffdb16a802fe2bb95e2e35ff34f0e53aeef34f",
 			"revisionTime": "2018-05-06T08:24:08Z"
 		},
-		{
```

Comment:
It appears you have removed the SDK from vendoring. We try to keep all dependencies up to date in the vendor folder.

Reply:
I have added it with  `govendor update +external github.com/newrelic/infra-integrations-sdk/...` and tested it locally